core/domain/wipeout_service.py

```diff
@@ -647,11 +697,17 @@ def _delete_models(user_id, module_name):
 
 
 def _collect_and_save_entity_ids_from_snapshots_and_commits(
-        pending_deletion_request,
-        activity_category,
-        snapshot_metadata_model_classes,
-        commit_log_model_class,
-        commit_log_model_field_name):
+    pending_deletion_request: wipeout_domain.PendingDeletionRequest,
+    activity_category: models.NAMES,
+    snapshot_metadata_model_classes: List[
+        Type[base_models.BaseSnapshotMetadataModel]
+    ],
+    commit_log_model_class: Optional[Type[base_models.BaseCommitLogEntryModel]],
```

Comment:
This looks odd to me. Why is this optional? Ditto for the next one.

/cc @vojtechjelinek

Reply:
Thanks for clearing the ambiguity!!

Reply:
I would suggest leaving a comment here to explain that, then.

Reply:
Done PTAL!